script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -75,7 +75,7 @@ For now, I'll express my intentions as best I can.
 "Should I call you Hisui then?"
 "Yes, no matter what."
 "...Haha."
-'No matter what' Hisui said strongly.
+'No matter what' Hisui said strongly. //I don't feel like this conveys the original line, 『なんとしても』という言い方がちょっと食い気味の翡翠だった, quite right. Could we maybe have a JP speaker check over it again?
```

Comment:
https://hinative.com/ja/questions/8872844
I guess the implication is that Hisui is very quick / vehement in her statement?

Reply:
idk maybe "sharply"?